Austin_Marga_HW7_2.ipynb

```diff
@@ -0,0 +1,189 @@
+{
```

Comment:
<p>padding, cutting, windowing would be useful in this step, to make <code>len(x,y)=2**x</code>  and also square!</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1_2DFFT/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>